build.gradle

```diff
@@ -253,6 +278,9 @@ subprojects {
         ballerinaStdLibs "org.ballerinalang:nats-ballerina:${stdlibNatsVersion}"
         ballerinaStdLibs "org.ballerinalang:system-ballerina:${stdlibSystemVersion}"
         ballerinaStdLibs "org.ballerinalang:runtime-ballerina:${stdlibRuntimeVersion}"
+        ballerinaStdLibs "org.ballerinalang:sql-ballerina:${stdlibSqlVersion}"
+        ballerinaStdLibs "org.ballerinalang:java.jdbc-ballerina:${stdlibJdbcVersion}"
+        ballerinaStdLibs "org.ballerinalang:mysql-ballerina:${stdlibMySqlVersion}"
```

Comment:
Shouldn't we need to specify version global variables like this?
```
stdlibSystemVersion = project.stdlibSystemVersion
```

Reply:
Is that necessary? The local build wee passing so I didn't add those.